dashboard/app/controllers/learning_goal_teacher_evaluations_controller.rb

```diff
@@ -0,0 +1,60 @@
+class LearningGoalTeacherEvaluationsController < ApplicationController
+  authorize_resource
+
+  def create
+    learning_goal_teacher_evaluation = LearningGoalTeacherEvaluation.new(learning_goal_teacher_evaluation_params)
+    learning_goal_teacher_evaluation.teacher_id = current_user.id
+
+    learning_goal_teacher_evaluation.save!
+
+    render json: learning_goal_teacher_evaluation
+  end
+
+  def update
+    params.require(:id)
+    learning_goal_teacher_evaluation = LearningGoalTeacherEvaluation.find_by_id(params[:id])
+
+    if learning_goal_teacher_evaluation&.teacher_id == current_user.id
+      if learning_goal_teacher_evaluation.update(learning_goal_teacher_evaluation_params)
+        render json: learning_goal_teacher_evaluation
+      else
+        head :bad_request
+      end
+    else
+      head :not_found
```

Comment:
I would expect CanCan to give you a 403 in this case before your controller method even executes, but your tests show that it returns not found. is there something I'm missing?